operator/pkg/util/util.go

```diff
@@ -140,3 +142,11 @@ func RenderTemplate(tmpl string, ts interface{}) (string, error) {
 	}
 	return buf.String(), nil
 }
+
+// BoolValue returns true if v is not nil and v.Value is true, or false otherwise.
+func BoolValue(v *v1alpha1.BoolValueForPB) bool {
```

Comment:
Can we remove this? You can just do v.GetValue() which is more clear and already nil-safe

Reply:
`v.GetValue()` may be not work, due to:
https://github.com/istio/api/blob/a9c5294cac41c9089557ef1464f8353c5ef2b346/operator/v1alpha1/common.go#L60-L62

If we choose to go that way, we may need to modify there.

Reply:
Added a new pr for this https://github.com/istio/api/issues/1938